frontend/src/v5/ui/controls/inputs/colorPicker/colorPicker.component.tsx

```diff
@@ -15,15 +15,15 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { useState, useEffect } from 'react';
+import { useState } from 'react';
 import ChevronIcon from '@assets/icons/outlined/thin_chevron-outlined.svg';
-import { defaults, isNumber } from 'lodash';
 import { ActionMenu } from '@controls/actionMenu';
 import { Container } from './colorPicker.styles';
 import { ColorPickerPalette } from './colorPickerPalette/colorPickerPalette.component';
 import { UNSET_RGB_COLOR } from './colorPicker.helpers';
 import { ColorCircle } from './colorCircle/colorCircle.styles';
-import { HexGroupColor, hexGroupColorToRgb, RgbGroupColor, rgbGroupColorToHex } from '@/v5/helpers/colors.helper';
+import { hexGroupColorToRgb, RgbGroupColor, rgbGroupColorToHex } from '@/v5/helpers/colors.helper';
+import { defaults } from 'lodash';
```

Comment:
When I click on the colour picker inside "edit/create group", it crashes

https://github.com/user-attachments/assets/a01db3c3-c590-42b9-b1a7-79ef7d441675